packages/blocks/blocksBasic/src/blocks/List/List.json

```diff
@@ -7,6 +7,18 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "direction": {
+          "type": "string",
+          "description": "List content as row or column."
```

Comment:
Can we please add some more detailed descriptions for all three of these properties? Since direction is implemented using `flex-direction`, I think it would be good to mention that here.

Reply:
Options should also be listed for enum properties.